Updates enterprise api page (#1242)
pomerium/documentation: content/docs/capabilities/enterprise-api.mdx

Reviewer: While you're here, would you mind updating line 82? I think the command there is outdated:
```console
$ go get github.com/pomerium/enterprise-client-go
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
```

As the error message says, I think this should be updated to:
```
go install github.com/pomerium/enterprise-client-go@latest
```

Author: Done.

## Configure a new route

1. We suggest configuring the route for API access in the open-source Pomerium Core. That way, changes made through the API that might break access to the Enterprise Console won't break access to the API route.

   ```yaml
   - from: https://console-api.localhost.pomerium.io
     to: https://pomerium-console-domain-name:8702
     pass_identity_headers: true
     allow_any_authenticated_user: true
     tls_custom_ca_file: /path/to/rootCA.pem
   ```

   :::info

>    See [**TLS Custom Certificate Authority**](/docs/reference/routes/tls#tls-custom-certificate-authority) for more information about this route setting.

Reviewer: I think it might be good to be explicit about what "this route setting" refers to.

```suggestion
   See [**TLS Custom Certificate Authority**](/docs/reference/routes/tls#tls-custom-certificate-authority) for more information about the `tls_custom_ca_file` setting.
```
